locale/uk_UA/editor.xml

```diff
@@ -2,15 +2,153 @@
 <!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
 
 <!--
-  * locale/uk_UA/editor.xml
+  * locale/en_US/editor.xml
   *
-  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2014-2018 Simon Fraser University Library
```

Comment:
All copyrights should be to Simon Fraser University, not Simon Fraser University Library.